waspc/run

```diff
@@ -25,20 +27,55 @@ BUILD_ALL_STATIC_CMD="$WASP_PACKAGES_COMPILE && $BUILD_HS_CMD --enable-executabl
 
 INSTALL_CMD="$WASP_PACKAGES_COMPILE && cabal install --overwrite-policy=always"
 
-TEST_UNIT_CMD="cabal test waspc-test"
-TEST_CLI_CMD="cabal test cli-test"
-TEST_E2E_CMD="cabal test e2e-test"
-TEST_E2E_DELETE_GOLDENS="rm -rf $PROJECT_ROOT/e2e-test/test-outputs/*-golden"
-TEST_E2E_ACCEPT_ALL_CMD="$TEST_E2E_DELETE_GOLDENS && $TEST_E2E_CMD"
-TEST_HEADLESS_CMD="cd $PROJECT_ROOT/examples/todoApp && npm install && npm run test"
-TEST_CMD="cabal test && echo 'Running headless tests' && $TEST_HEADLESS_CMD && echo 'ALL TESTS PASSED' || { echo 'SOME TESTS FAILED'; exit 1; }"
+# Testing
+WASPC_UNIT_TESTS_CMD="cabal test waspc-tests"
+WASPC_E2E_TESTS_CMD="cabal test waspc-e2e-tests"
```

Comment:
I talked about this in comments on the README - the terminology we communicate to users does not entirely match the terminology in in the script/command/file system.